cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterTwillApplication.java

```diff
@@ -342,9 +341,7 @@ private void prepareLogSaverResources(Path tempDir, CConfiguration containerCCon
       Path libJar = Files.createTempFile("log.lib", ".jar");
       try (JarOutputStream jarOutput = new JarOutputStream(Files.newOutputStream(libJar))) {
         for (File jarFile : libJars) {
-          JarEntry jarEntry = new JarEntry(jarFile.getName());
-          jarEntry.setMethod(ZipEntry.STORED);
```

Comment:
It needs to provide the file size and crc (which needs to be computed). Anyway it just simpler to do this.